### ISSUE TITLE
[COGNITION] Pittsburgh Sidenav - Links lack sufficient styling to differentiate from text

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#3426]
## Description
**[COGNITION] Pittsburgh Sidenav - Links lack sufficient styling to differentiate from text**
Color must not be used as the only way to distinguish links from regular text. [WCAG 1.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html)
![pitt-sidenav-link-color-only.png](https://images.zenhubusercontent.com/5d6685f1a77bcf04870bc81c/832e76f3-034c-4dd0-8978-15990cbb6452)

## Type of Issue
Template

## Point of Contact

Jennifer

## Acceptance Criteria

As a website user (particularly for low-vision and color blind), I want to distinguish links from regular text content so I may access interactive content.

## Environment

* Operating System: all
* Browser: all

## Steps to Recreate

1. Enter `https://staging.va.gov/pittsburgh-health-care/` in browser
2. Discern links from regular text by more than color difference

## Possible Fixes (optional)

Follow [design.va.gov](https://design.va.gov/design/typography#links) link styling to include an underline style for links.
![pitt-sidenav-typography-links.png](https://images.zenhubusercontent.com/5d6685f1a77bcf04870bc81c/09c054b9-f146-4f45-abdf-54dd6c761aa7)

**Comment by jenstrickland** [comment]
After discussion with @rtwell we decided the latest styling is sufficient. As always, usability testing with people who may be particularly vulnerable to misunderstanding will be helpful to ensure we're 👍 .
